services/storage/src/simcore_service_storage/db_file_meta_data.py

```diff
@@ -71,9 +71,11 @@ async def list_filter_with_partial_file_id(
     project_ids: list[ProjectID],
     file_id_prefix: str | None,
     partial_file_id: str | None,
+    only_files: bool,
 ) -> list[FileMetaDataAtDB]:
     stmt = sa.select(file_meta_data).where(
-        (
+        (file_meta_data.c.is_directory == (not only_files))
```

Comment:
is this a boolean then use:

```suggestion
        (file_meta_data.c.is_directory.is_(not only_files))
```
or
```suggestion
        (file_meta_data.c.is_directory.isnot_(only_files))
```